frontend/apps/mobile/src/app/pages/page-create-tournament/page-create-tournament.component.ts

```diff
@@ -0,0 +1,221 @@
+
+import { tournamentDetailsSelector } from '@jtr/business-domain/tournament';
+import { TournamentData } from '@jtr/data-domain/store';
+import { SingletonGetter } from '@jtr/infrastructure/cache';
+
+import { createTournamentFormControl } from '../../../../../../libs/business-domain/tournament/src/lib/form-controls/create-tournament-form.control';
+import { PageCreateTournamentInformationAccommodationComponent } from './page-create-tournament-information-accommodation/page-create-tournament-information-accommodation.component';
+import { PageCreateTournamentInformationAdditionalComponent } from './page-create-tournament-information-additional/page-create-tournament-information-additional.component';
+import { PageTournamentInformationBasicComponent } from './page-create-tournament-information-basic/page-create-tournament-information-basic.component';
+import { PageCreateTournamentInformationContactComponent } from './page-create-tournament-information-contact/page-create-tournament-information-contact.component';
+import { PageCreateTournamentInformationRegistrationComponent } from './page-create-tournament-information-registration/page-create-tournament-information-registration.component';
+import { PageCreateTournamentInformationRulesComponent } from './page-create-tournament-information-rules/page-create-tournament-information-rules.component';
+import { PageCreateTournamentSubmitComponent } from './page-create-tournament-submit/page-create-tournament-submit.component';
+import { TranslatePipe } from '@ngx-translate/core';
+import { DividerModule } from 'primeng/divider';
+import { TournamentService } from '../../../../../desktop/src/app/business-rules/tournament/tournament.service';
+
+@Component({
+  standalone: true,
+  imports: [
+    CommonModule,
+    PageTournamentInformationBasicComponent,
+    PageCreateTournamentInformationAccommodationComponent,
+    PageCreateTournamentInformationContactComponent,
+    PageCreateTournamentInformationRegistrationComponent,
+    PageCreateTournamentInformationRulesComponent,
+    PageCreateTournamentInformationAdditionalComponent,
+    PageCreateTournamentSubmitComponent,
+    ReactiveFormsModule,
+    DividerModule,
+    TranslatePipe,
+  ],
+  providers: [DatePipe],
+  templateUrl: './page-create-tournament.component.html',
+  styleUrl: './page-create-tournament.component.less',
+})
+export class PageCreateTournamentComponent implements OnDestroy {
+  public form = createTournamentFormControl;
+  private readonly destroy$ = new Subject<void>();
+
+  private newTournament = true;
+
+  @SingletonGetter()
+  public get tournament$(): Observable<TournamentData | null> {
+    return this.store$.select(tournamentDetailsSelector);
+  }
+
+  constructor(
+    private tournamentService: TournamentService,
+    private router: Router,
+    private readonly store$: Store,
+    private readonly datePipe: DatePipe
+  ) {
+    this.tournament$.pipe().subscribe((tournament) => {
```

Comment:
Hier fehlt das: `takeUntil(this.destroy$)`
(Bei der desktop Variante btw. auch)